README.md

```diff
@@ -1,6 +1,16 @@
 # motion-controls
 
 
+## Installation
+
+### Linux (Ubuntu-like)
+
+1. Install godot (*Caution*: The flatpak version does *not* work) `sudo apt install godot3`
```

Comment:
I am unsure about this. The following almost works except for the *libhidapi*:
`flatpak run --env=LD_LIBRARY_PATH=$(pwd)/JoyCons org.godotengine.Godot -e`
I am guessing it just needs another `LIBRARY_PATH`